bin/oref0-setup.sh

```diff
@@ -503,6 +505,10 @@ fi
 echo Running: openaps report add enact/suggested.json text determine-basal shell monitor/iob.json monitor/temp_basal.json monitor/glucose.json settings/profile.json $EXTRAS
 openaps report add enact/suggested.json text determine-basal shell monitor/iob.json monitor/temp_basal.json monitor/glucose.json settings/profile.json $EXTRAS
 
+# create .profile so that openaps commands can be executed from the command line
```

Comment:
added TODO and created issue https://github.com/openaps/oref0/issues/299 for eliminate the environement variables